addons/mass_mailing/static/src/js/tours/mass_mailing_saving_tour.js

```diff
@@ -0,0 +1,23 @@
+odoo.define('mass_mailing.mass_mailing_tour', function (require) {
```

Comment:
Would name tour ``mass_mailing.mailing_tour_editor`` or something like that (by the way, not sure what it really tests, could be great to add some explanations)

Reply:
Wow you're quick, this commit is called just "test" because it's not finished, give me a minute :rofl: 
Thanks anyway for the help :-D

Reply:
Ow, ok :D as I saw @xmo-odoo reviewed, I fourred my truffe in it :)